### ISSUE TITLE
FTB Inferno 1.10.0 crashes with shader activated!

### ISSUE THREAD

**Issue by Brosfight** [FTBTeam/FTB-Modpack-Issues#4457]
### Modpack

FTB Inferno

### Modpack version

1.10.0

### Log Files

[crash-2023-12-12_09.56.41-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13645970/crash-2023-12-12_09.56.41-client.txt)
[crash-2023-12-12_09.35.38-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13645971/crash-2023-12-12_09.35.38-client.txt)
[crash-2023-12-12_10.02.33-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13645973/crash-2023-12-12_10.02.33-client.txt)

### Describe the bug

Game crashes with shaders. The provided one or any other shader. 
The crash log description isn't the same every time.
The moment you active a shader, the game crashes. I tried it on a Server and Singleplayer World. I tried the older Modpack version 1.9.1 and everything works fine.
Im running a Ryzen 7800X3D and a RTX 4090 with 32GB Ram.
I allocated 8Gb to the MC Instance.
It is only occurring with the newest 1.10.0 Modpack version.
I reinstalled the Modpack around 5-6 Times now. It is always the same.
All settings are at default. I only deactivated vsync and unlocked the max FPS.
All my drivers are uptodate (nvidia,windows,mainboard,chipset,even bios)

### Steps to reproduce

Install newest Modpack version 1.10.0
Activate the provided shader (while being in the menu or in a loaded world)
Game crashes

### Expected behaviour

To be able to play with a shaderpack (provided or not)

### Screenshots

![Screenshot 2023-12-12 095942](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/39033087/def52515-7b06-450f-8f3a-e642c56767e2)
![Screenshot 2023-12-12 095958](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/39033087/fadd8ee8-bcbe-4961-9f2d-866485bd10e4)
![Screenshot 2023-12-12 100010](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/39033087/021aa8e4-e2b4-41a3-9030-fc0c42eeb892)
![Screenshot 2023-12-12 100026](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/39033087/cd9046c4-ed4a-412a-9a12-0842a5e4178b)

### Additional information

The crashlogs shows a different entity every time the game crashes.

**Comment by Exovi** [comment]
I am having the same issue :-( Guess downgrading is the only solution

**Comment by Snoddas76** [comment]
Same issue for me too, testing different shaders.

**Comment by Slytherite** [comment]
yeah same here and upon reviewing the crashlog, it crashed trying to render Foxhounds within the shader.

**Comment by kszesek** [comment]
I think I found a way to fix the crash. replace the oculus version that's in the pack with oculus-mc1.18.2-1.6.4

**Comment by Slytherite** [comment]
Yeah, I just did that and shaders started working without crash!
